src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartRpc.cs

```diff
@@ -30,8 +33,30 @@ public StartRpc(INethermindApi api)
         public async Task Execute(CancellationToken cancellationToken)
         {
             IJsonRpcConfig jsonRpcConfig = _api.Config<IJsonRpcConfig>();
+            IKeyStoreConfig keyStoreConfig = _api.Config<IKeyStoreConfig>();
             ILogger logger = _api.LogManager.GetClassLogger();
 
+            if (string.IsNullOrEmpty(jsonRpcConfig.JwtSecretFile))
+            {
+                string oldPath = "keystore/jwt-secret";
+                string newPath = Path.Join(keyStoreConfig.KeyStoreDirectory, "jwt-secret");
+                // check if jwt-secret file already exists in previous default directory
+                if (!File.Exists(newPath) && File.Exists(oldPath))
+                {
```

Comment:
This improves log output with the full path instead of `keystore/jwt-secret`:
```suggestion
                {
                    oldPath = Path.GetFullPath(oldPath);
                    newPath = Path.GetFullPath(newPath);
```

Reply:
I'd just go ahead and commit these